openxr/examples/vulkan.rs

```diff
@@ -409,27 +441,40 @@ fn main() {
                 continue;
             }
 
+            // Block until the previous frame is finished displaying, and is ready for another one.
+            // Also returns a prediction of when the next frame will be displayed, for use with
+            // predicting locations of controllers, viewpoints, etc.
             let xr_frame_state = frame_wait.wait().unwrap();
+            // Must be called before any rendering is done!
             frame_stream.begin().unwrap();
 
             if !xr_frame_state.should_render {
                 frame_stream
                     .end(
                         xr_frame_state.predicted_display_time,
-                        xr::EnvironmentBlendMode::OPAQUE,
+                        environment_blend_mode,
                         &[],
                     )
                     .unwrap();
                 continue;
             }
 
             if swapchain.is_none() {
+                // Now we need to find all the viewpoints we need to take care of! This is a
+                // property of the view configuration type; in this example we use PRIMARY_STEREO,
+                // so we should have 2 viewpoints.
+                //
+                // Because we are using multiview in this example, we will only look at the first
+                // viewpoint, and assume that the other viewpoints are identical.
```

Comment:
```suggestion
                // Because we are using multiview in this example, we require that all view
                // dimensions are identical.
```